src/main/java/uk/gov/hmcts/cmc/claimstore/services/ClaimantResponseService.java

```diff
@@ -164,23 +165,52 @@ private void formaliseResponseAcceptance(
         }
     }
 
-    private void raiseAppInsightEvents(Claim claim, Response response, ClaimantResponse claimantResponse) {
+    private void raiseAppInsightEvents(Claim claim,
+                                        Response response,
+                                        ClaimantResponse claimantResponse,
+                                        CaseEvent caseEvent) {
```

Comment:
to conform to de-facto standard
```suggestion
    private void raiseAppInsightEvents(
        Claim claim,
        Response response,
        ClaimantResponse claimantResponse,
        CaseEvent caseEvent
    ) {
```